Analyzer/include/Config.h

```diff
@@ -257,7 +275,7 @@ class Config
             leptonFileName                    = "allInOne_leptonSF_UL.root";
             hadronicFileName                  = "allInOne_hadronicSF_UL.root";
             bjetFileName                      = "allInOne_BTagEff_UL.root";
-            bjetCSVFileName                   = "wp_deepCSV_106XUL17_v3.csv";
+            bjetCSVFileName                   = "wp_deepJet_106XUL17_v3.csv";
```

Comment:
Can we rename these variables to remove usage of "CSV", just to avoid any confusion ?

Reply:
Resolved in 6f5d619